workshops/gloo-mesh-demo/README.md

```diff
@@ -54,12 +54,13 @@ Set these environment variables which will be used throughout the workshop.
 ```sh
 # Used to enable Gloo Mesh (please ask for a trail license key)
 export GLOO_MESH_LICENSE_KEY=<licence_key>
-export GLOO_MESH_VERSION=v2.1.0-beta29
+export GLOO_MESH_VERSION=v2.1.0-rc2
 
 # Istio version information
 export ISTIO_IMAGE_REPO=us-docker.pkg.dev/gloo-mesh/istio-workshops
-export ISTIO_IMAGE_TAG=1.13.8-solo
-export ISTIO_VERSION=1.13.8
+export ISTIO_IMAGE_TAG=1.15.1-solo
```

Comment:
i dont think we can deploy 1.15 yet? we dont have the multi cluster routing fix in or we can wait for that to land

Reply:
Reverted back until issue is fully resolved.